r_examples/r_serving_with_fastapi/FastAPI Example.ipynb

```diff
@@ -0,0 +1,2155 @@
+{
```

Comment:
<p>Same comments as other notebook - please use AWS-hosted Iris dataset and cite.</p><p> </p><p>Be aware that if the files in this example are updated in the future, all the commits are hard-coded in your links, so the links will still refer to the original version. They can be updated, but usually are overlooked in revisions. Are you using the commit links in case content is moved or deleted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2902/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I am using relative hyperlinks in the markdown cells. Is that what you are referring to?

Reply:
You can disregard the link feedback - ReviewNB shows the link as, for instance, `https://github.com/aws/amazon-sagemaker-examples/raw/765161bdded598c4c83bc30cb1ccfa823e27ac3d/r_examples/r_serving_with_plumber` when you specify `../r_serving_with_plumber`. No problem to leave as-is with relative links.